leanpkg.toml

```diff
@@ -5,3 +5,4 @@ lean_version = "leanprover-community/lean:3.50.3"
 path = "src"
 
 [dependencies]
+mathlib = {git = "https://github.com/leanprover-community/mathlib", rev = "290a7ba01fbcab1b64757bdaa270d28f4dcede35"}
```

Comment:
This doesn't make any sense in mathlib itself!

Reply:
Removed.